frontend/src/components/InfoPanel.jsx

```diff
@@ -223,7 +225,7 @@ class InfoPanel extends React.PureComponent {
                   <Histogram
                     isoCode={this.props.isoCode}
                     height={this.props.height}
-                    dateInitial={this.state.dateModelStart}
+                    dateInitial={this.state.dateFirstCase}
                     dateFinal={this.state.dateModelEnd}
```

Comment:
Should `dateFinal` use `dateFirstWaveEnd` instead of `dateModelEnd`? Or possibly we should be taking `min` or `max` between these?

Reply:
I think dateFirstWaveEnd is the same as dateModelEnd (28 day after end of lockdown), but yeah, it does make sense that the histogram has `dateFirstWaveEnd` as input.

Reply:
Ok, so `dateFirstWaveEnd` is not part of the Infopanel state (is part of the Country Statistics one), but both are indeed the same variable, coming from `maximum_date` in the `loadrealdates` task.

I could rename it in infopanel so the variable name makes more sense but then the `dateModelEnd` variable is not going to be paired with anything. I think i'll leave as it is for now.